ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Common/screen.cpp

```diff
@@ -45,6 +45,9 @@ void scene_end() {
 }
 
 unsigned char* graphics_copy_screen_pixels(int x, int y, int width, int height, bool* flipped) {
+  #ifndef GL_READ_FRAMEBUFFER
+    return nullptr;
```

Comment:
Don't return nullptr, below this there's an array allocation. Check the macro and return the array there without calling the unsupported function. This will keep the game from crashing at the very least, even if it doesn't exactly add the feature.